w3-aggregation.md

```diff
@@ -0,0 +1,293 @@
+
+## Authorization
+
+A Filecoin broker authority (e.g. Spade) MUST give capabilities to Storefront principals (e.g. web3.storage) to invoke capabilities to get stored content into Filecoin deals.
+
+```mermaid
+sequenceDiagram
+    participant Authority as 🌐<br/><br/>did:web:spade.storage
+    participant Storefront as 🌐<br/><br/>did:web:web3.storage
+
+    Authority->>Storefront: delegate `aggregate/*`
+```
+
+Note that this can be done out of band with presentation of a delegation signature.
+
+## Deal creation
+
+When a Storefront has enough content to fulfill an aggregate (each broker might have different requirements), a Filecoin deal for an aggregate SHALL be requested by a `aggregate/submit` invocations. Deal negotiations with Filecoin Storage Providers will be handled out off band. Therefore, Broker should generate a receipt to acknowledge received request. This receipt contains an [effect](https://github.com/ucan-wg/invocation/#7-effect) to be performed (`.fx` field pointing to another task) that will eventually get a receipt with either `ok` or `error`.
+
+```mermaid
+sequenceDiagram
+    participant Storefront as 🌐<br/><br/>did:web:web3.storage
+    participant Authority as 🌐<br/><br/>did:web:spade.storage
+
+    Storefront->>Authority: invoke `aggregate/submit`
+    Note left of Authority: Prepare for deal
+    Authority-->>Storefront: receipt
+    Note left of Authority: Land deal in Filecoin
+```
+
+## Deal information
+
+Storefront users MAY want to check details about deals from the content they previously stored. These deals will change over time as they get renewed. Therefore, Storefront should invoke `aggregate/deals` capability to gather information about given aggregate identifier.
+
+```mermaid
+sequenceDiagram
+participant Storefront as 🌐<br/><br/>did:web:web3.storage
+    participant Authority as 🌐<br/><br/>did:web:spade.storage
+
+    Storefront->>Authority: invoke `aggregate/deals`
+```
+
+## Capabilities
+
+This section describes the capabilities that form the w3 aggregation protocol, along with the details relevant for invoking capabilities with a service provider.
+
+In this document, we will be looking at `spade.storage` as an implementer of the `aggregate` and `deals/*` protocol.
+
+### `aggregate/*`
+
+A Filecoin broker Authority MAY delegate capabilities to any Storefront principal.
+
+> `did:web:spade.storage` delegates `aggregate/*` capabilities to `did:web:web3.storage`
+
+```json
+{
+  "iss": "did:web:spade.storage",
+  "aud": "did:web:web3.storage",
+  "att": [{"with": "did:web:spade.storage", "can": "aggregate/*" }],
+  "exp": null,
+  "sig": "..."
+}
+```
+
+### `aggregate/submit`
+
+A Storefront principal can invoke a capabilty to submit an aggregate ready for deals.
+
+> `did:web:web3.storage` invokes capability from `did:web:spade.storage`
+
+```json
+{
+  "iss": "did:web:web3.storage",
+  "aud": "did:web:spade.storage",
```

Comment:
I am personally not excited ( because I hate `deal` as a concept in general, and have phased out that language within spade entirely ). However @dkkapur is running the client-facing show, so deferring to him :)

Reply:
What the term used by spade ?